### PR TITLE
Document MakeApiBaselines script

### PR DESCRIPTION
**Pull request by sebastienros**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/extensions/pull/4526)

### CODE REVIEW

File: scripts/MakeApiBaselines.ps1 [hunk 1]
```diff
@@ -4,6 +4,12 @@
     Creates API baseline files representing the current API surface exposed by this repo.
 #>
 
+if ($PSVersionTable.PSVersion.Major -lt 6) {
```
Comment (sebastienros) [inline scripts/MakeApiBaselines.ps1:7]: Required for the `-LeafBase` argument. Even though I had a recent windows install, my PS version was 5.x

**Review by RussKie — APPROVED** [review]

**Comment by geeknoid** [comment]
@joperezr Why was this merged? I have a comment in there saying that this content is incorrect.

**Comment by sebastienros** [comment]
@geeknoid I don't see any comment, maybe you forgot to hit the "submit review" in GH?

**Comment by joperezr** [comment]
I don't see any comment.. 🤔

**Comment by geeknoid** [comment]
HMmm, my comment was marked as Pending for some reason. I pushed it now...

File: docs/building.md [hunk 2]
```diff
@@ -148,7 +148,8 @@ This sets up necessary environmental variables and opens the repository in VS Co
 * Windows Forms uses Visual Studio MSBuild but for certain features we require the latest MSBuild from .NET Core/.NET SDK. If you are on an official version of [Visual Studio][VS-download] (i.e. not a Preview version), then you may need to enable previews for .NET Core/.NET SDKs in VS.
   * you can do this in VS under Tools->Options->Environment->Preview Features->Use previews of the .Net Core SDK (Requires restart)
 * When restoring packages, if you get the following message: "The SSL connection could not be established, see inner exception. Unable to read data from the transport connection", try disabling IPv6 on your network adapter.
-
+* When the public API surface is updated, you need to update the API baselines manifest files for the impacted assemblies. [Here is an example](https://github.com/dotnet/extensions/blob/3e6ac0735cb62d4c8b63e131d956397a87ea26f7/src/Libraries/Microsoft.Extensions.AsyncState/Microsoft.Extensions.AsyncState.json) for the `Microsoft.Extensions.AsyncState` project. To update the manifest of all the local assemblies, invoke the script `./scripts/MakeApiBaselines.ps1`.
```
[the remark this answers is not in the document]
Comment (sebastienros) [reply docs/building.md:151]: Do you suggest I remove the comment completely or wait for November to do it? I believe we get warnings currently if we don't run the script.

**Pull request merged**